prism-languages.php

```diff
@@ -4,6 +4,7 @@
  * Retrieve support languages
  * @return array Hash of lang => language
  */
+// Test PHPCS
 function mkaz_code_syntax_block_get_supported_languages() {
```

Comment:
:no_entry_sign: **Error**: You must use &quot;/**&quot; style comments for a function comment (*Squiz.Commenting.FunctionComment.WrongStyle*).